.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
must be added in .stylelintrc https://stylelint.io/user-guide/rules/indentation#options